ibis/backends/postgres/tests/test_postgis.py

```diff
@@ -71,16 +68,20 @@ def test_select_multipolygon_geodata(geotable):
 
 
 def test_geo_area(geotable, gdf):
+    import geopandas
```

Comment:
shouldn't you have a skip if this is not available? (as its not a strict requirement of postgres)

Reply:
Those skips are what caused the tests to be silently skipped. Not a big fan of those, they've been causing to tests not run without anyone noticing several times both in pandas and in ibis. So, this approach doesn't skip silently, and uses markers to control what is being run. If dependencies are not present tests will fail. If we don't want those tests to run, because we don't want to test this, we can just call pytest with `-m "not geo"`. I think this is very simple, and we won't have these tests being skipped from the CI again for months without noticing.

I'd like to eventually have all these skips removed.

Reply:
ok cool.